packages/manager/modules/backup-agent/src/utils/mappers/mapTenantToTenantWithAzName.ts

```diff
@@ -0,0 +1,13 @@
+import {Vault} from "@/types/Vault.type";
+import {Resource, ResourceWithAzName} from "@/types/Resource.type";
+import {Tenant} from "@/types/Tenant.type";
+
+export type TenantWithName = Pick<Tenant, "name">
+export type TenantWithVault = Pick<Tenant, "vaults">
+
+export const mapTenantToTenantWithAzName = <T extends TenantWithVault>(tenant: T): T & Pick<ResourceWithAzName["currentState"], "azName"> =>
+  ({...tenant, azName: tenant.vaults[0]?.azName ?? ''});
+
+export const mapTenantResourceToTenantResourceWithAzName = <T extends TenantWithVault & TenantWithName>(resourceTenant: Resource<T>): Resource<T> & ResourceWithAzName =>
+  ({...resourceTenant, currentState: mapTenantToTenantWithAzName(resourceTenant.currentState) });
```

Comment:
These lines are crazy long so I think there is something off with the linter configuration. 
You might better fix this sooner than later